src/Data/Either/Plucky.hs

```diff
@@ -30,7 +30,7 @@ module Data.Either.Plucky
     , catchT
     , catchOneT
       -- * Concise Type Signature Helper
-    , OneOf(..)
+    , OneOf
```

Comment:
Not really part of this PR, but ghcide was moaning at me. It's also complaining that you defined `rethrowT` but aren't actually using it...